envi/archs/i386/emu.py

```diff
@@ -16,9 +16,21 @@
 
 def shiftMask(val, size):
     if size == 1:
-        return (val & 0x1f) % 9
+        return (val & 0x1f) % 8
     elif size == 2:
-        return (val & 0x1f) % 17
+        return (val & 0x1f) % 16
+    elif size == 4:
+        return val & 0x1f
+    elif size == 8:
+        return val & 0x3f
+    else:
+        raise Exception("shiftMask is broke in envi/arch/i386/emu.py")
+
+def shiftMaskRC(val, size):
```

Comment:
So you added this, but you don't seem to call it anywhere?

Reply:
this is mentioned below :)  not wired in because i think i have a faster impl inline.  but you say you'd rather use it, so i'll wire it in.

Reply:
Either works, I'd prefer this one since it's technically "more" correct. But if you're not going to use it, I'd prefer you to delete this so we don't have vestigial code just kinda lying around that we have no idea in 6 months why it's still there.

Reply:
totally agreed.  you prefer using it, so i'll wrap it in.  :)
it's in there for the conversation.